### PR TITLE
Update to support Ember 3.28 and 4.4

### PR DESCRIPTION
**Pull request by mikrostew**
Updates this addon to work with Ember@3.28 and Ember@4.4.

I used `ember-cli-update` to get this working with 3.28 first, then again to convert to 4.4.

`yarn lint`, `yarn test`, and `yarn test:all` are working for me locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ As noted in Paul Irish's "[What Forces Layout](https://gist.github.com/paulirish
 
 ## Compatibility
 
-- Ember.js v3.8 or above
-- Ember CLI v2.13 or above
-- Node.js v10 or above
+* Ember.js v3.24 or above
+* Ember CLI v3.24 or above
+* Node.js v12 or above
```
Comment (rwjblue) [inline README.md:17]: Can you bump this to v14+? (It doesn't make sense to me to do a major release and still support an already EOL'ed Node version)

File: config/ember-try.js [hunk 2]
```diff
@@ -1,55 +1,49 @@
...
         npm: {
           devDependencies: {
-            'ember-source': '~3.12.0',
+            'ember-source': '~3.24.3',
           },
         },
       },
       {
-        name: 'ember-lts-3.16',
+        name: 'ember-lts-3.28',
```
Comment (rwjblue) [inline config/ember-try.js:19]: Can you also add `ember-lts-4.4` and `ember-lts-4.8` (both here and in ci.yml) now that they are both LTS?

File: package.json [hunk 3]
```diff
@@ -41,60 +45,71 @@
...
+    "qunit-dom": "^2.0.0",
     "release-it": "^12.2.1",
     "release-it-lerna-changelog": "^1.0.3",
-    "typescript": "^3.5.3"
+    "typescript": "^4.8.4",
+    "webpack": "^5.72.1"
   },
   "engines": {
-    "node": "10.* || >= 12.*"
+    "node": "12.* || 14.* || >= 16"
```
Comment (rwjblue) [inline package.json:105]: Please update to `14.* || 16.* || >= 18` (node 12 is already EOL'ed)

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mikrostew) [reply README.md]: fixed in 9b060a19a1e4698862945f731c2a3f257233ab9a

File: config/ember-try.js (hunk 2 above)
Comment (mikrostew) [reply config/ember-try.js:19]: fixed in 9b060a19a1e4698862945f731c2a3f257233ab9a

File: package.json (hunk 3 above)
Comment (mikrostew) [reply package.json]: fixed in 9b060a19a1e4698862945f731c2a3f257233ab9a

**Review by scalvert — APPROVED** [review]

**Pull request merged**